templates/project/.editorconfig

```diff
@@ -14,10 +14,10 @@ trim_trailing_whitespace = true
 [*.{yaml,yml,twig,php}]
 indent_size = 4
 
-[*.{js,json,scss,css}]
-indent_size = 2
+[.yamllint]
```

Comment:
```suggestion
[.yaml]
```

?

Reply:
@core23 there really is a `.yamllint` file in YAML format, I think this is intended.

Reply:
yes it's intended